src/main/java/seedu/clinkedin/ui/ResultDisplay.java

```diff
@@ -13,11 +13,28 @@ public class ResultDisplay extends UiPart<Region> {
 
     private static final String FXML = "ResultDisplay.fxml";
 
+    private static final String DEFAULT_INIT_DISPLAY = "Welcome to CLInkedIn!\n\n"
+            + "CLInkedIn is a desktop contact management application for Recruiters to "
+            + "manage their contact list of candidates.\n"
+            + "1. Keep track of the candidates/contacts you have met through events or recruitment applications.\n"
+            + "2. Tag candidates by their skills, experience and stage in the recruitment process.\n"
+            + "3. Filter candidates according to a specific skill, experience or stage in the recruitment process.\n"
```

Comment:
Would find be a better word to use instead of filter, since that is our feature name?